package.json

```
@@ -7,7 +7,10 @@
   "scripts": {
     "lint": "yarn run lint:js && yarn run lint:scss",
     "lint:js": "standardx 'app/assets/javascripts/**/*.js' 'spec/javascripts/**/*.js'",
-    "lint:scss": "stylelint app/assets/stylesheets/"
+    "lint:scss": "stylelint app/assets/stylesheets/",
+    "jasmine:prepare": "bundle exec rake assets:precompile",
+    "jasmine:ci": "yarn run jasmine:prepare && PATH=~/.webdrivers:$PATH yarn run jasmine-browser-runner runSpecs",
```

```suggestion
    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
```

This suggestion is for consistency with our other apps: https://github.com/alphagov/content-publisher/blob/6511019bc2313b1cea3f1fcd7f5e77aef6913368/package.json#L12

This app doesn't have the webdrivers gem (it was removed from GOV.UK test) https://github.com/alphagov/govuk_test/pull/43